wsgi.py

```diff
@@ -0,0 +1,37 @@
+from flask import Flask
+from flask.logging import default_handler
+
+import os
+import logging
+
+import s3
+import producer
+
+application = Flask(__name__)
+
+ROOT_LOGGER = logging.getLogger()
+ROOT_LOGGER.setLevel(application.logger.level)
+ROOT_LOGGER.addHandler(default_handler)
+
+@application.route("/", methods=['GET', 'POST']) # using 'GET' temporarily to trigger the microservice by hitting the route in the browser
+def wake_up():
+    server = os.environ.get('KAFKA_SERVER')
+    topic = os.environ.get('KAFKA_TOPIC')
+
+    aws_key = os.environ.get('AWS_ACCESS_KEY_ID')
+    aws_secret = os.environ.get('AWS_SECRET_ACCESS_KEY')
+    aws_bucket = os.environ.get('AWS_S3_BUCKET_NAME')
+
+    application.logger.info('Saving data to bucket %s', aws_bucket)
+    filesystem = s3.connect(aws_key, aws_secret)
+    s3.save_data(filesystem, aws_bucket, "Data going to bucket")
+
+    value = {'url': "INPUT DATA ORIGIN IN S3", 'rh_account': "ACCOUNT"}
```

Comment:
Let's use our custom format instead. Don't place random strings in random fields. Let's rather do something like:
```suggestion
    value = {
        'message': 'Data published to s3',
        'origin': 'AI-Ops',
        'url': f's3:/{aws_bucket}/<SOME_DESCRIPTION_PLEASE_REPLACE>'
    }
```

Reply:
I wonder if we need `<SOME_DESCRIPTION_PLEASE_REPLACE>` ...

`f's3:/{aws_bucket}'` should suffice for the url, right?